### PR TITLE
Export keras.losses.{huber_loss, logloss}

### PR DESCRIPTION
**Pull request by danaugrs**
See https://github.com/tensorflow/tensorflow/issues/26012#issuecomment-466645654

### CODE REVIEW

File: tensorflow/python/keras/losses.py [hunk 1]
```diff
@@ -810,6 +815,8 @@ def logloss(y_true, y_pred):
   return K.mean(-losses, axis=-1)
 
 
+@keras_export('keras.losses.huber',
+              'keras.losses.huber_loss')
 def huber_loss(y_true, y_pred, delta=1.0):
```
Comment (pavithrasv) [inline tensorflow/python/keras/losses.py]: Can we update the name to huber and just export `keras.losses.huber`?

Comment (danaugrs) [reply tensorflow/python/keras/losses.py]: Done 😄

File: tensorflow/python/keras/losses.py [hunk 2]
```diff
@@ -801,7 +801,12 @@ def categorical_hinge(y_true, y_pred):
   return math_ops.maximum(0., neg - pos + 1.)
 
 
+@keras_export('keras.losses.logloss')
```
Comment (fchollet) [inline tensorflow/python/keras/losses.py]: This is the same as binary crossentropy. Shouldn't we remove all instances of "logloss" from the API? (preferring binary_crossentropy instead).

Comment (danaugrs) [reply tensorflow/python/keras/losses.py]: Good point. I removed the logloss function and class entirely since I don't think we want to keep those. Let me know if you want any more changes! 😄

Comment (pavithrasv) [reply tensorflow/python/keras/losses.py]: https://github.com/tensorflow/tensorflow/commit/55240a22f10a9a6cf3cf8ba3d35e16d88f5925f6#diff-b789cb2a4dba80832969100ce22623cb removed logloss. Please sync to the latest changes.

Comment (danaugrs) [reply tensorflow/python/keras/losses.py]: Ok - synced with latest.

**Review by pavithrasv — DISMISSED** [review]

**Comment by rthadur** [comment]
@danaugrs can you fix the build failures : https://source.cloud.google.com/results/invocations/33e50490-71a6-4de6-96e9-2fcf3683026d/log

**Comment by danaugrs** [comment]
@rthadur I fixed it - needs to be reviewed again apparently @pavithrasv.

File: tensorflow/python/keras/losses.py [hunk 3]
```diff
@@ -871,7 +872,11 @@ def sparse_categorical_crossentropy(y_true, y_pred, from_logits=False, axis=-1):
 
 @keras_export('keras.metrics.binary_crossentropy',
               'keras.losses.binary_crossentropy')
-def binary_crossentropy(y_true, y_pred, from_logits=False, label_smoothing=0):  # pylint: disable=missing-docstring
+def binary_crossentropy(y_true, y_pred, from_logits=False, label_smoothing=0):
+  """Computes the binary cross-entropy between `y_true` and `y_pred`.
+
+  `binary_crossentropy = - y_true*log(y_pred) - (1 - y_true)*log(1 - y_pred)`
```
Comment (pavithrasv) [inline tensorflow/python/keras/losses.py]: Missed this last time, can we change this to `loss =` for consistency with other losses.

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by rthadur** [comment]
@pavithrasv can you please review new changes.

File: tensorflow/python/keras/losses.py [hunk 4]
```diff
@@ -871,7 +872,11 @@ def sparse_categorical_crossentropy(y_true, y_pred, from_logits=False, axis=-1):
 
 @keras_export('keras.metrics.binary_crossentropy',
               'keras.losses.binary_crossentropy')
-def binary_crossentropy(y_true, y_pred, from_logits=False, label_smoothing=0):  # pylint: disable=missing-docstring
+def binary_crossentropy(y_true, y_pred, from_logits=False, label_smoothing=0):
+  """Computes the binary cross-entropy between `y_true` and `y_pred`.
```
Comment (pavithrasv) [inline tensorflow/python/keras/losses.py]: Can we remove this and keep the change minimal to just exporting huber loss?

**Review by pavithrasv — COMMENTED** [review]
Can we update the PR title to just have huber loss?

**Comment by pavithrasv** [comment]
Closing the PR due to inactivity. Please feel free to reopen.
